src/main/java/trackitnus/logic/commands/contact/EditContactCommand.java

```diff
@@ -71,8 +71,8 @@ private static Contact createEditedContact(Contact contactToEdit, EditContactDes
         assert contactToEdit != null;
 
         Name updatedName = editContactDescriptor.getName().orElse(contactToEdit.getName());
-        Phone updatedPhone = editContactDescriptor.getPhone().orElse(contactToEdit.getPhone());
-        Email updatedEmail = editContactDescriptor.getEmail().orElse(contactToEdit.getEmail());
+        Phone updatedPhone = editContactDescriptor.getPhone().orElse(contactToEdit.getPhone().orElse(null));
+        Email updatedEmail = editContactDescriptor.getEmail().orElse(contactToEdit.getEmail().orElse(null));
```

Comment:
What if the contact originally has a phone number and the user wants to edit to remove it. This will ensure that if the user types `C edit 1 p/` the phone number will remain unchanged when it should be changed to null

Reply:
Fixed! For the commit I just pushed to fix this I have manually tested deleting phones & emails and it all worked well!